src/apis/accesscontextmanager/v1beta.ts

```diff
@@ -411,13 +411,9 @@ export namespace accesscontextmanager_v1beta {
      */
     unrestrictedServices?: string[] | null;
     /**
-     * Beta. Configuration for within Perimeter allowed APIs.
+     * Beta. Configuration for APIs allowed within Perimeter.
```

Comment:
Eugh.  Could I trouble you to first submit and land a re-generate PR from master?  Would love to see only the relevant changes in this PR.  Sorry.